.github/workflows/win.yml

```diff
@@ -37,7 +37,7 @@ defaults:
 env:
   BUILD_TYPE: Release
   MSVC_VERSION: '2022'
-  VCPKG_VERSION: '2024.01.12'
+  VCPKG_VERSION: '8040303'
```

Comment:
newest commit on vcpkg, the "old" release failed to resolve libspatialite with pkg-config for some reason (worked until this PR and I didn't change anything regarding pkg-config or spatialite..)